InSimDotNet/Packets/IS_MAL.cs

```diff
@@ -13,8 +12,9 @@ namespace InSimDotNet.Packets
     /// You can set a list of up to 120 mods that are allowed to be used on a host
     /// Send zero to clear the list and allow all mods to be used
     /// </remarks>
-    public class IS_MAL : IPacket
+    public class IS_MAL : IPacket, ISendable
     {
+        public static int MAX_MAL_MODS = 120;
```

Comment:
this should probably be const or at least readonly

Reply:
I agree. Changed to const